obs-studio-server/source/nodeobs_api.cpp

```diff
@@ -487,14 +483,20 @@ static void                                    node_obs_log(int log_level, const
         levelname.length(),
         levelname.c_str());
 #endif
-	if (length < 0)
+	if (length < 0) {
+		std::lock_guard<std::mutex> lock(logMutex);
+		outdated_driver_error::instance()->catch_error(msg);
```

Comment:
I don't think this is necessary in this case

Reply:
OK. I just tried to keep the same execution path for all cases.

Reply:
Yup makes sense but yeah ok to remove

Reply:
OK